canonical-websites/insights.ubuntu.com
# Remove bower dependencies

Removed bower dependencies.

# QA

- `./run clean` 
- `./run`
- `http://0.0.0.0:8023`
- check that you don't have any error in your console
- `./run clean`
- Check that the folders `/static/css/modules/` and `/static/js/modules/` is empty (except the `.gitkeep` file)

# Issue

I had to manually publish the package `cookie-policy`, so I opened this issue: https://github.com/canonical-webteam/cookie-policy/issues/10
